app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -474,15 +462,22 @@ private void takeScreenshotOfErrorPages(UiDevice device) throws UiObjectNotFound
 
             TestHelper.inlineAutocompleteEditText.setText("error:"+ error.value);
             device.pressKeyCode(KEYCODE_ENTER);
-
-            onWebView()
-                    .withElement(findElement(Locator.ID, "errorTitle"))
-                    .perform(webClick());
-
-            onWebView()
-                    .withElement(findElement(Locator.ID, "errorTryAgain"))
-                    .perform(webScrollIntoView());
-
+            TestHelper.webView.waitForExists(waitingTime);
+
+            if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
onWebView() fails on API 26

Reply:
Do you have more information here? Is this a bug in espresso that might get fixed in a later version? I wonder why you are not checking for 26 then?

Reply:
It looks like on API 26, this Locator.ID isn't found anymore. (up to API 25, it seems to work according to BB run results) On the other hand, the UiObject definition works on all APIs, so I supect it could be deprecated somehow?